src/Microsoft.Spatial/System/Collections/Generic/ReadOnlyCollectionExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace System.Collections.Generic
+{

```

Comment:
any reason why do we put this extension into the Microsoft.Spatial lib?

Reply:
I have moved the extensions to `Microsoft.OData.Edm` to have the same effect but keep `Microsoft.Spatial` unchaged.